ruby/vampires.rb

```diff
@@ -0,0 +1,57 @@
+puts "How many employees will be processed?"
+employees_to_process = gets.chomp
+
+
+employees_to_process.to_i.times do
+  puts "What is your name?"
+  name = gets.chomp
+  puts "How old are you?"
+  age = gets.chomp
+  puts "What year were you born?"
+  birth_year = gets.chomp
+  puts "Do you enjoy Garlic bread? yes/no"
+  garlic = gets.chomp
+  puts "Would you like to enroll in company health insurance? yes/no"
+  insurance = gets.chomp  
+
+  allergy = nil
+  allergies = Array.new
+
+  puts "Please list any allergies. Type 'done' when finished."
+
+  until allergy == "done" || allergy == "sunshine" do     
```

Comment:
The way this reads there will always be a string `"done"` in the allergies array.

Do you want all of the people you are interviewing to be allergic to done?